src/actions/index.js

```diff
@@ -0,0 +1,20 @@
+import createRoom from '../socketHandlers/createRoom';
+import io from 'socket.io-client';
+const socket = io('localhost:9000/game');
+
+const createGame = () => {
+  return async (dispatch) => {
+    await createRoom();
```

Comment:
I think this `await` doesn't have any meaning because `createRoom` function doesn't return a Promise object. (return nothing.)